CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Next
 * Added Travis-CI - [@dblock](https://github.com/dblock).
 * Added TODO list and enhanced README documentation - [@ptitvert](https://github.com/ptitvert)
 * [#15](https://github.com/dblock/oshi/pull/15), [#18](https://github.com/dblock/oshi/pull/18): Added support for CPU load - [@kamenitxan](https://github.com/kamenitxan), [@Sorceror](https://github.com/Sorceror).
+* Include inactive memory in mac/linux available - [@dbwiddis](https://github.com/dbwiddis).
```

Comment:
It's an opportunity!

We force this strange construction where you commit, then push, then make a pull request, then amend. It's weird, but it's the best we have for changelog entries that seems to make the customer (the developer reading the changelog) happy.